data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1228 @@
+			`	Despite the aggressive action, you are not followed as you leave the planet.`
+				flee
+
+
+
+mission "Gegno Asteroid Mining Prologue"
+	landing
+	name "Mine 5 tons of iron"
+	description "Mine and return 5 tons of iron to <destination>."
+	source "Tschyss"
+	to offer
+		has "Acquiescence: offered"
+		not "Gegno Genocide Defense: offered"
+	on offer
+		conversation
+			branch "returned"
+				has "Gegno Asteroid Mining Prologue: deferred"
+
+			`You arrive at your destination: a massive space station. Instead of standing directly behind you in your cockpit this time, the Gegno ended up taking a seat next to one of your open bunks, and didn't move a muscle up until now. Despite the grandiose nature of the Gegno's one and only space station, it seems very old and inactive. As you are escorted around the spacious interior, of which a majority is only a skeletal frame of pipes, bars, and supportive structure, you see a noticeable lack of personnel and running machinery. Some sections aren't even properly pressurized, but, thankfully, your spacesuit helps with that. Occasionally, you come across some workers, but they appear very lethargic, or maybe just unmotivated.`
+			`	Eventually, you end up in a large docking bay where more workers and ships are present. Some of the workers are hauling crates of materials back and forth, with others talking amongst each other. The Gegno with you then motions over to a half-cracked display board on one of the walls across the room. With one last meaningful look, the Gegno leaves you to your own devices. You wonder if you'll ever see them again, considering the way they've been treated by the others.`
+			choice
+				`	(Go to the display.)`
+					goto "board"
+				`	(Try to communicate with the other Gegno around you.)`
+			
+			`	You attempt to get the attention of some of the other Gegno present in the facility, but most of them give you strange glares or outright ignore you, until one seemingly annoyed Gegno gestures to the large cracked display.`
+
+			label "board"
+			`	The display, although cracked and beaten up, still appears otherwise functional. It presents a series of graphics in a list that seems to be updating in real-time. On the left side of each graphic are various types of mineable asteroids, and on the right are tally marks, which thankfully you can understand. It appears to be some sort of job list linked with the station, which implies that the Gegno are at least giving you a chance to work with them here, to "do your part", or perhaps "make up for what you've done."`
```

Comment:
The part about understanding is already implied in the last sentence.
```suggestion
			`	The display, although cracked and beaten up, still appears otherwise functional. It presents a series of graphics in a list that seems to be updating in real-time. On the left side of each graphic are various types of mineable asteroids, and on the right are tally marks. It appears to be some sort of job list linked with the station, which implies that the Gegno are at least giving you a chance to "do your part", or perhaps "make up for what you've done."`
```